proposals/0026-hlsl-long-vector-type.md

```diff
@@ -0,0 +1,278 @@
+<!-- {% raw %} -->
+
+# HLSL Long Vectors
+
+* Proposal: [0026-HLSL-Vectors](0026-hlsl-vector-type.md)
+* Author(s): [Anupama Chandrasekhar](https://github.com/anupamachandra), [Greg Roth](https://github.com/pow2clk)
+* Sponsor: [Greg Roth](https://github.com/pow2clk)
+* Status: **Under Consideration**
+
+## Introduction
+
+HLSL has always supported vectors of as many as four elements of different element types (int3, float4, etc.).
```

Comment:
Always doesn't indicate enough that this is past behavior. and `as many as` just read awkward to me. Also replaced the repeat of the word element with `scalar`.
```suggestion
HLSL has traditionally supported vectors with up to four elements of different scalar types (int3, float4, etc.).
```

Reply:
This comment went to the wrong suggestion. Oops